articles/mariadb/index.md

```diff
@@ -26,17 +26,17 @@ In this article you will learn how to;
 
 - fetch the saved data from the Database  
 
-### Step 1: Installing Xampp
+## Step 1: Installing Xampp
 
 Xampp comes along with PhpMyAdmin when downloaded into a machine.  PhpMyAdmin is the MariaDatabase that we will be working on within this article.
 
 To install Xampp you need to;
 
-### . Download the package to be installed.
+## . Download the package to be installed.
```

Comment:
No need for this section instead go straight to the contents. Remove the heading.